src/actor.cxx

```diff
@@ -2,19 +2,27 @@
 
 #include "actor.h"
 
-// TODO(pottumuusi) Do not allow to construct without name
-Actor::Actor() {
-    std::cout << "Default constructor of Actor running" << std::endl;
+Actor::~Actor()
+{
+    std::cout << "Destructor of Actor running" << std::endl;
 }
 
-Actor::~Actor() {
-    std::cout << "Destructor of Actor running" << std::endl;
+Actor::Actor(std::string _name)
```

Comment:
Duplicated implementation. Call own more detailed constructor instead. Same issue in player.cxx. Fix in: (bb43fe9)